src/helpers/handleError.ts

```diff
@@ -29,5 +30,5 @@ export default function (error: unknown) {
   }
   if (!displayedError) displayedError = ErrorList.unknown
 
-  toast.error(displayedError)
+  toast.error(transformRelayErrorMessage(displayedError))
```

Comment:
Can we parse error before toasting and reassign `displayedError`? If we'll need another parser to connect, we'll need to wrap this message again, that's not good